spec/controllers/subject_rule_effects_controller_spec.rb

```diff
@@ -3,27 +3,125 @@
 RSpec.describe SubjectRuleEffectsController, type: :controller do
   let(:workflow) { create :workflow }
   let(:rule) { create :subject_rule, workflow: workflow }
+  let(:credentials) do
+    fake_session admin: false, project_ids: [workflow.project_id], logged_in: true
+  end
 
   context 'as a permissioned user' do
-    before{ fake_session admin: false, project_ids: [workflow.project_id], logged_in: true }
+    before { credentials }
+
+    describe '#create' do
+      let(:create_params) do
+        {
+          subject_rule_effect: { action: 'retire_subject', config: {} },
+          workflow_id: workflow.id,
+          subject_rule_id: rule.id
+        }
+      end
+
+      it 'does not create a new effect' do
+        post :create, params: create_params, format: :json
+        expect(response.status).to eq(401)
+      end
+
+      it 'redirects to the new path in html mode' do
+        post :create, params: create_params, format: :html
+        expect(response).to redirect_to(
+          new_workflow_subject_rule_subject_rule_effect_path
+        )
+      end
+
+      it 'flashes an error message' do
+        post :create, params: create_params, format: :html
+        expect(flash[:alert]).to eq('Error creating a subject effect rule')
+      end
+
+      xit 'makes a new effect' do
+        post :create, params: {subject_rule_effect: {action: 'retire_subject', config: {}}, workflow_id: workflow.id, subject_rule_id: rule.id }, format: :json
+
+        expect(response.status).to eq(201)
+        result = JSON.parse(response.body)
+        expect(result["id"]).not_to be(nil)
+        expect(result["subject_rule_id"]).to eq(rule.id)
+      end
+
+      xit 'redirects to the subject rule in html mode' do
```

Comment:
see above, the x isn't super obvious and we don't wanna forget

Reply:
right - these are pending specs that we will revisit once we circle back to this
https://relishapp.com/rspec/rspec-core/v/3-9/docs/pending-and-skipped-examples/skip-examples#temporarily-skipping-by-prefixing-%60it%60,-%60specify%60,-or-%60example%60-with-an-x

Reply:
would you like a combined comment?
```suggestion
      # TODO: fix these pending specs when revisiting admin lockdown
      xit 'redirects to the subject rule in html mode' do
```

Reply:
I got that, I'm just recommending that the lil `x` might deserve a TODO so we don't miss it when we come back.